src/thunderbots/software/test_util/BUILD

```diff
@@ -0,0 +1,18 @@
+package(default_visibility = ["//visibility:public"])
+
+cc_library(
+    name = "test_util",
+    testonly = 1,
```

Comment:
should this be used for all tests?

Reply:
Not quite sure what you mean here. Do you mean the `test_util` should be linked into all tests? There are some tests that don't use it? Or are you confused about what the `testonly` flag does? It basically specifies that this library may only be linked into test executables.

Reply:
Seems like the testonly flag should be used more

Reply:
I'm happy to use it, just couldn't find any other libraries that should only ever be built into tests. Let me know if you see any!

Reply:
I'll look into it but it's not blocking